client/src/Components/Cards.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+const Cards = ({img,title,description}) => {
+    return (
+        <div className='flex flex-col w-[250px] m-6 border-2 border-black text-center p-2 rounded-lg'>
```

Comment:
can you please put a comment to documentation where all possible options of `className` are listed, it is a bit hard to imagine what is being selected here in this `div` tag?

Reply:
It is `tailwind`, in which we add styling in our classes rather than making separate style sheet for CSS.
Which of-course will make a question on the readability of our code.

Reply:
please add `Tailwind` documentation link defining styling attributes.